### PR TITLE
[v3] Fix default context-menu in selected disabled inputs

### PR DESCRIPTION
**Pull request by mmghv**
`processDefaultContextMenu()` was missing showing the default context-menu for selected text in a disabled or readonly input, this PR fixes that to be aligned with v2.

### CODE REVIEW

File: v3/examples/contextmenus/main.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func main() {
 	mainWindow := app.NewWebviewWindowWithOptions(application.WebviewWindowOptions{
 		Title:  "Context Menu Demo",
 		Width:  1024,
-		Height: 1124,
+		Height: 1024,
```
Comment (leaanthony) [inline v3/examples/contextmenus/main.go:31]: I set this to 1124 as all the boxes didn't fit on Windows :-) Do they all fit in Mac?

Comment (leaanthony) [reply v3/examples/contextmenus/main.go:31]: image

Comment (leaanthony) [reply v3/examples/contextmenus/main.go:31]: I just sized it and then ran `wails.Window.Size()` in devtools and got this:
```
height: 1681,
width:1024
```

**Comment by leaanthony** [comment]
I accidentally pushed some updates to your PR branch 😅 It's all good - we can merge.

File: v3/examples/contextmenus/main.go (hunk 1 above)
Comment (mmghv) [reply v3/examples/contextmenus/main.go:31]: On my screen a little bit was hidden under the windows taskbar, so made it smaller to fit and scrollbars with appear, I don't know how it looks on Mac.

Comment (leaanthony) [reply v3/examples/contextmenus/main.go:31]: Ah the joys of different screen resolutions 😅

**Pull request merged**